src/mpid/ch4/shm/posix/eager/stub/stub_init.h

```diff
@@ -13,13 +13,13 @@
 
 #include "stub_impl.h"
 
-MPL_STATIC_INLINE_PREFIX int MPIDI_POSIX_eager_init(int rank, int size)
+int MPIDI_POSIX_eager_init(int rank, int size)
 {
```

Comment:
Changes for `stub_init` should be in a separate commit.

Reply:
• I have used the netmod as blueprint for the uninline. If we don't move `init|finalize` to a separate .c file these will be undefined at link time when inline is disabled;
• To split the `init|finalize` functions from the rest of the inlined functions as noted in previous point.

There might be a better way of doing this. I have tried to follow the netmod for consistency within the code.

Reply:
Should this commit change .h to .c?